docs/guides.md

```diff
@@ -1061,213 +1057,244 @@ The same procedure may be used to install custom KEK, db, or dbx certs.
 
 Note that the virtual firmware (uefistored + OVMF), as is allowed by the specification, does not mandate that these default certificates be signed by their parent (i.e., the KEK doesn't need to be signed by PK) if they're installed via `secureboot-certs`. This verification *does* occur, however, when trying to enroll new certificates from inside the guest after boot. This is designed to give the host administrator full control over the certificates from the control domain.
 
-#### Install an Archived UEFI DBX
-
-The UEFI Forum hosts archived DBX auth files [here](https://uefi.org/revocationlistfile/archive).
-
-Download the x64 archive manually, and then provide the path to the file to `secureboot-certs`
-for the `dbx` argument:
+If necessary for your use case you may omit the `dbx` entirely. Note that this basically **renders secure boot useless** from a security perspective, as any binary signed with a revocated certificate will still pass Secure Boot checks! This may be done by using the following command:
```

Comment:
I had to look up the term "revocated" because it sounded a little off to me. It appears that some dictionaries do consider it "obsoleted" (https://gcide.gnu.org.ua/?q=revocate&define=Define&strategy=. and apparently the Oxford English Dictionary appear to). Perhaps "revoked" may sound more natural?

revocated is used in a total of 12 places here, so if this changes they probably should too.

Reply:
I trust you on this, I only used "revocated" because it seemed to me it went along naturally with "revocation". Fixed everywhere.